toml.md

```diff
@@ -677,6 +692,17 @@ extraneous whitespace.
 [ j . "ʞ" . 'l' ]  # same as [j."ʞ".'l']
 ```
 
+Indentation is treated as whitespace. Tabs and spaces before keys are ignored.
+
+```toml
+[m]      # how to write subtables
+[m.n]    # use dotted keys to nest tables
+[m.n.o]  # best practice: write all headers flush-left
+
+[p]      # Indentation is VALID,
+  [p.q]    # BUT DISCOURAGED.
```

Comment:
I would remove this whole example, since it gives only stylistic advice. Not our cup of tea!

Reply:
Alright.